### ISSUE TITLE
Doesn't run in Python 2.6

### ISSUE THREAD

**Issue by martinthurn-LFT** [c-w/Gutenberg#21]
Your code doesn't work in Python 2.6.  Please add a warning in your documentation.

**Comment by c-w** [comment]
Hi Martin,

Thanks for reporting this issue. I'll have a look if I can make the code run under Python 2.6 and add supported-versions disclaimer to the documentation otherwise.

Yours,
Clemens

**Comment by c-w** [comment]
Setting up a Python 2.6 development environment:
```bash
sudo add-apt-repository ppa:fkrull/deadsnakes
sudo apt-get update
sudo apt-get install python2.6

virtualenv -p "$(which python2.6)" venv-2.6
. venv-2.6/bin/activate
pip install -r requirements.pip
pip install nose
```

It looks like there are three main reasons why the tests are failing:
- String formatting requires explicitly named or numbered placeholders.
- The `gzip.open` function doesn't work in the `with` statement.
- The `unittest.TestCase` class lacks a bunch of assertion methods.

These problems are all quite easy to fix and don't affect the code quality much, so I'm happy to fix the issues and make the project Python 2.6 compatible.
